index.js

```diff
@@ -81,10 +82,13 @@ grunt.initConfig({
   ci: {
     buildNumber: process.env.BUILD_NUMBER || 0,
     buildId: process.env.BUILD_ID || 0,
+    buildBranch: process.env.BRANCH_NAME || "",
+    buildRevision: process.env.CHANGE_TITLE || "",
```

Comment:
Can we remove or rename `buildRevision`?

- "Revision" is often used interchangeably with "Commit" or the "Build Number", usually a unique ID. Here, `buildRevision` refers to the name/title of the Pull Request, and I think that may be unexpected.

- It isn't being used in the manifest. In the manifest `Build-Revision` is using `buildNumber`.

Reply:
done